### PR TITLE
Seperate asset service from market service (final)

### PR DESCRIPTION
**Pull request by elmutt**
Final changes needed for separate use of asset service and market service.

Type changes, name changes, remove stuff from market service and new asset lookup function

### CODE REVIEW

**Review by DaoDev44 — APPROVED** [review]

File: packages/market-service/package.json [hunk 1]
```diff
@@ -20,6 +20,7 @@
     "test": "jest test"
   },
   "dependencies": {
+    "@shapeshiftoss/asset-service": "^1.0.0",
```
Comment (0xdef1cafe) [inline packages/market-service/package.json:23]: uhhh we can't be doing this or we end up with coupled packages

Comment (0xdef1cafe) [reply packages/market-service/package.json:23]: @cjthompson ?

**Review by 0xdef1cafe — CHANGES_REQUESTED** [review]

**Review by theoboldfrazier — APPROVED** [review]
LGTM

File: packages/market-service/package.json (hunk 1 above)
Comment (cjthompson) [reply packages/market-service/package.json:24]: As long as there isn't a circular dependency it's fine.

File: packages/asset-service/src/generateAssetData/generateAssetData.ts [hunk 2]
```diff
@@ -20,6 +20,10 @@ const generateAssetData = async () => {
     `./src/service/generatedAssetData.json`,
     JSON.stringify(generatedAssetData)
   )
+
+  const assetService = new AssetService()
+
+  await assetService.initialize()
```
Comment (cjthompson) [inline packages/asset-service/src/generateAssetData/generateAssetData.ts:26]: I don't see `assetService` being called anywhere else.

File: packages/market-service/package.json (hunk 1 above)
Comment (cjthompson) [reply packages/market-service/package.json:24]: However, it does look to me like it doesn't actually do anything with asset-service.

Also, we should be using our DI model which would take an AssetService in the constructor on not have a direct dependency on the package.

File: packages/market-service/src/coingecko/coingecko.ts [hunk 3]
```diff
@@ -1,19 +1,11 @@
 import axios from 'axios'
 import dayjs from 'dayjs'
-import { MarketService, AssetMarketData, HistoryData, HistoryTimeframe } from '../api'
+import { MarketService, MarketData, HistoryData, HistoryTimeframe } from '../api'
+import { ChainTypes } from '@shapeshiftoss/asset-service'
```
Comment (cjthompson) [inline packages/market-service/src/coingecko/coingecko.ts:4]: I think we should move shared types out into a `@shapeshiftoss/types` package

File: packages/asset-service/src/generateAssetData/generateAssetData.ts (hunk 2 above)
Comment (elmutt) [reply packages/asset-service/src/generateAssetData/generateAssetData.ts:26]: good call. was using this for testing. i can remove it

File: packages/market-service/src/coingecko/coingecko.ts (hunk 3 above)
Comment (elmutt) [reply packages/market-service/src/coingecko/coingecko.ts:4]: Agreed. I want to do that in another PR so I can get this merged before people continue using the old market service for everything and make more changes

**Review by 0xdef1cafe — APPROVED** [review]

**Pull request closed without merging**